### PR TITLE
GTM README updates

### PR DESCRIPTION
**Pull request by tunetheweb**
Add link to https://github.com/google-marketing-solutions/web-vitals-gtm-template

### CODE REVIEW

**Comment by tunetheweb** [comment]
FYI @demirj

**Comment by demirj** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -363,9 +363,10 @@ For details on how to query this data in [BigQuery](https://cloud.google.com/big
...
+While `web-vitals` can be called directly from Google Tag Manager, using a pre-defined custom template makes this considerably easier. Some recommended templates include:
 
-For full installation and usage instructions, see Simo's post: [Track Core Web Vitals in GA4 with Google Tag Manager](https://www.simoahava.com/analytics/track-core-web-vitals-in-ga4-with-google-tag-manager/).
+- [Simo Ahava](https://www.simoahava.com/) has created and maintains a [Core Web Vitals](https://www.simoahava.com/custom-templates/core-web-vitals/). For full installation and usage instructions, see Simo's post: [Track Core Web Vitals in GA4 with Google Tag Manager](https://www.simoahava.com/analytics/track-core-web-vitals-in-ga4-with-google-tag-manager/).
+- The Google Marketing Solutions team have created [their GTM own template](https://github.com/google-marketing-solutions/web-vitals-gtm-template), including support for attribution data, and loading the `web-vitals` library from [unpkg.com](https://unpkg.com) or locally.
```
Comment (philipwalton) [inline README.md:369]: I'm concerned that there's too much editorializing in the current text. Also I wouldn't list specific features as those could change (e.g. Simo could add attribution support).

Can you reformat these bullets to be something like:

- [Templated Name] by [creator]: See [resource name] for usage and installation instructions.

**Review by philipwalton — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tunetheweb) [reply README.md]: There was a fair amount of editorializing in there already ;-)

But fair enough - done.

**Review by philipwalton — APPROVED** [review]

**Pull request merged**